src/rust/wpcomics/sources/nettruyen/src/lib.rs

```diff
@@ -6,62 +6,50 @@ use aidoku::{
 	prelude::*,
 	std::net::Request,
 	std::String,
-	std::{defaults::defaults_get, Vec},
+	std::{Vec},
 	Chapter, DeepLink, Filter, FilterType, Listing, Manga, MangaPageResult, MangaViewer, Page,
 };
 use wpcomics_template::{helper::urlencode, template, template::WPComicsSource};
 
+const BASE_URL: &str = "https://nettruyenvia.com";
+
 fn get_instance() -> WPComicsSource {
 	WPComicsSource {
-		base_url: String::from("https://www.nettruyenvv.com"),
+		base_url: String::from(BASE_URL),
 		next_page: "li.active + li > a[title*=\"kết quả\"]",
 		viewer: MangaViewer::Rtl,
 		listing_mapping: |listing| {
 			String::from(match listing.as_str() {
-				"Truyện con gái" => "truyen-con-gai",
-				"Truyện con trai" => "truyen-con-trai",
-				"Hot" => "hot",
+				"Truyện con gái" => "truyen-tranh-con-gai",
+				"Truyện con trai" => "truyen-tranh-con-trai",
+				"Hot" => "truyen-tranh-hot",
 				_ => "",
 			})
 		},
 		status_mapping: status_map,
 		time_converter: convert_time,
+		manga_viewer_page_attr: "data-src",
 		page_url_transformer: |url| {
-			let mut server_two = String::from("https://images2-focus-opensocial.googleusercontent.com/gadgets/proxy?container=focus&gadget=a&no_expand=1&resize_h=0&rewriteMime=image%2F*&url=");
-			if let Ok(server_selection) = defaults_get("serverSelection") {
-				if let Ok(2) = server_selection.as_int() {
-					server_two.push_str(&urlencode(url));
-					server_two
-				} else {
-					url
-				}
-			} else {
-				url
-			}
+			url
```

Comment:
you don't need to provide the `page_url_transformer` field if you don't use it (it has a default value)